packages/gitlab/src/api/GitlabCIClient.ts

```diff
@@ -138,6 +138,7 @@ export class GitlabCIClient implements GitlabCIApi {
                 Record<string, unknown>[]
             >('users', {
                 search: contributorsData[i].email,
+                without_project_bots: true,
```

Comment:
Can you substitute it with: `without_project_bots: 'true'`

Reply:
Will do! I am on the move today so will update the PR later today, thanks for all your help :smile:

Reply:
All done! :+1: